docs/getting-started/users-rewards.md

```diff
@@ -6,6 +6,63 @@ sidebar_label: Rewards
 
 ## How do you get DHX tokens from the DHX DAO's Unlocked Reserves?
 
-Refer to the <a href="https://www.mxc.org/mxcdatadash" target="_blank" className="pretty-link pretty-link-colored">MXC DataDash App</a>.
 
-The DHX DAO Unlocked Reserves that will be governed by the DHX DAO Treasury will have 30% of the total supply of DHX tokens at the genesis block and other ongoing fees. These funds will be used to reward Proof of Participation by users in governance, mining, and roaming.
+The DHX DAO Unlocked Reserves (that will be governed by the DHX DAO Treasury) will have 30% of the total supply of DHX tokens at the genesis block and other ongoing fees. These funds will be used to distribute rewards though DataHighway Proof of Participation consensus mechanism by users in governance and mining.
+
+## How to get rewards participating on the DHX blockchain?
+
+DHX is a polkadot parachain, and every 12 seconds, one block is added to the blockchain by a winner Collator Node. Once the Collator wins, the rewards are distributed between the Collator Node and their Delegators:
+
+![Collator-Delegator](../../static/img/tutorial/Collator-Delegator.png)
```

Comment:
Change this line to:
![Collator-Delegator](/img/tutorial/Collator-Delegator.png)